assets/src/scss/layout/_navbar.scss

```diff
@@ -196,6 +213,7 @@ $navbar-default-height: 60px;
     @include large-and-up {
       display: flex;
       flex-grow: 1;
+      align-items: center;
       justify-content: end;
```

Comment:
Not really related to the Donate button menu but can be quickly solved, here it should be `flex-end` instead of `end` (which doesn't work in Safari)

Reply:
Actually it looks like `end` is [not even a valid value](https://css-tricks.com/almanac/properties/j/justify-content/) for this property, it's lucky that it works in some browsers 😄